src/main/drivers/compass_hmc5883l.h

```diff
@@ -17,7 +17,7 @@
 
 #pragma once
 
-#include "drivers/io.h"
```

Comment:
I think that every step to a better state is worth taking, but if we want the includes to include the directory prefix then some of this PR is going the other way. So I suggest that we at least do not make ot worse by removing the directory prefixes in this PR. - That said we do not need to add missing prefixes in all includes.